experiment.py

```diff
@@ -0,0 +1,29 @@
+from preprocess import get_data
```

Comment:
seems like we don't need this script. could this also be git ignored?

Reply:
okay